app/cdap/components/hydrator/components/Canvas/PipelineDiffNode.tsx

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import { Handle, Position } from 'reactflow';
+import classnames from 'classnames';
+import styled from 'styled-components';
+
+const StyledDiv = styled.div`
+  height: 100px;
+  width: 200px;
+  border: 2px solid black;
+  border-radius: 5px;
+  background: white;
+  z-index: 2;
+
+  label {
```

Comment:
I'm also unsure why you included this css like this? why not just make a label styled component? Also why are you using label here? Semantically the html label tag normally refers to an input or something like that